src/main/haskell/kore/app/exec/Main.hs

```diff
@@ -96,6 +97,7 @@ data KoreExecOptions = KoreExecOptions
     , isKProgram          :: !Bool
     -- ^ Whether the pattern file represents a program to be put in the
     -- initial configuration before execution
+    , smtTimeout          :: !Integer
```

Comment:
`sbv` calls the time-out `TimeOut` not `Timeout`; we should be consistent with that.